### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by FodT**
fix module name in npm install command
examples were causing some confusion since they were not copy-pastable.

### CODE REVIEW

**Comment by FodT** [comment]
resolves #50

File: README.md [hunk 1]
```diff
@@ -56,9 +56,12 @@ Retrieve, edit and save a single entity
 ``` js
 var agencyPromise = new t1.Entity('agency')
   .get(1234, connection)
-  .then(function(agency) {this.agency = agency});
-agency.name = 'new name';
-agency.save(connection).done(console.log('saved'));
+  .then(function(agency) {
+    this.agency = agency;
```
Comment (seanhan-mediamath) [inline README.md]: I think it's better to skip this? If 'use strict' is on, then 'this' will be undefined. Just a little simpler without.

File: README.md [hunk 2]
```diff
@@ -56,9 +56,12 @@ Retrieve, edit and save a single entity
...
+    agency.name = 'new name';
+    agency.save(connection).then(function () {
```
Comment (seanhan-mediamath) [inline README.md]: super nitpicky, but feels cleaner  as an example to return this promise and have .then in the same chain as the .get and first .then? 

var agencyPromise = new t1.Entity('agency')
    .get('1234', connection)
    .then(function(agency){
        agency.name = 'new name'
        return agency.save(connection)
    })
    .then(function() {
      console.log('saved')
    })
    .catch(error => console.log(error))

File: README.md [hunk 3]
```diff
@@ -69,10 +72,12 @@ Returns a generator to entities
 var userParams = {
   'page_limit':10
   };
-t1.EntityList.get('campaigns', connection,  userParams).then(function(list) {this.pg1 = list});
-t1.EntityList.getNextPage(pg1, connection).then(function(list) {this.pg2 = list});
-
-for (var entity of pg1.entities) {console.log(entity)}
+t1.EntityList.get('campaigns', connection,  userParams).then(function(list) {
+  this.pg1 = list;
```
Comment (seanhan-mediamath) [inline README.md]: same comment re: use of 'this', feels less confusing to skip it for the exampe

**Pull request merged**